### PR TITLE
Fix windows installer build

### PR DESCRIPTION
**Pull request by n1hility**
Workaround problem with PanelSwExt.

[NO NEW TESTS NEEDED]

```release-note
none
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Windows installer build failed, it looked like a network error.  I've restarted.
LGTM once the tests are hip.  Thanks for looking into this so quickly @n1hility

File: contrib/win-installer/build.ps1 [hunk 1]
```diff
@@ -12,8 +12,8 @@ function FetchPanel() {
     $ProgressPreference = 'SilentlyContinue'
     Invoke-WebRequest -UseBasicParsing -OutFile nuget.exe -ErrorAction Stop `
         -Uri https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
-
-    .\nuget.exe install PanelSwWixExtension
+    # 3.3.3.224 has issue, so fix to 211 for now
```
[the remark this answers is not in the document]
Comment (n1hility) [reply contrib/win-installer/build.ps1]: Good point! I updated the description

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**